vm/src/stdlib/winapi.rs

```diff
@@ -177,16 +227,18 @@ mod _winapi {
         args: CreateProcessArgs,
         vm: &VirtualMachine,
     ) -> PyResult<(usize, usize, u32, u32)> {
-        let mut si = winbase::STARTUPINFOEXW::default();
+        let mut si = Threading::STARTUPINFOEXW::default();
         si.StartupInfo.cb = std::mem::size_of_val(&si) as _;
 
         macro_rules! si_attr {
```

Comment:
For using tuple struct, we need to modify original macro like below. Please improve it as you want
```rust
macro_rules! si_attr {
            ($attr:ident, $t:ty) => {{
                si.StartupInfo.$attr = <Option<$t>>::try_from_object(
                    vm,
                    args.startup_info.get_attr(stringify!($attr), vm)?,
                )?
                .unwrap_or(0)
            }};
            ($attr:ident, $t:ty, $tt:path) => {{
                si.StartupInfo.$attr = $tt(<Option<$t>>::try_from_object(
                    vm,
                    args.startup_info.get_attr(stringify!($attr), vm)?,
                )?
                .unwrap_or(0));
            }};
        }
        si_attr!(dwFlags, u32, STARTUPINFOW_FLAGS);
        si_attr!(wShowWindow, u16);
        si_attr!(hStdInput, isize, HANDLE);
        si_attr!(hStdOutput, isize, HANDLE);
        si_attr!(hStdError, isize, HANDLE);
```